packages/react/src/field/FieldContainer.tsx

```diff
@@ -3,7 +3,9 @@ import { Stack } from '../box';
 import { boxPalette } from '../core';
 
 export type FieldContainerProps = PropsWithChildren<{
```

Comment:
For consistency, I've added prop comments here and exported the type while I was touching these files.